package.json

```diff
@@ -1,82 +1,76 @@
 {
-  "name": "generator-brei-app",
-  "version": "3.2.4",
-  "description": "Yeoman generator",
-  "license": "MIT",
-  "main": "generators/app/index.js",
-  "repository": "BarkleyREI/generator-brei-app",
-  "bugs": "https://github.com/BarkleyREI/generator-brei-app/issues",
-  "author": {
-    "name": "Ian Moffitt",
-    "email": "nessthehero@gmail.com",
-    "url": "https://github.com/nessthehero"
-  },
-  "contributors": [
-    {
-      "name": "Ian Moffitt",
-      "email": "nessthehero@gmail.com",
-      "url": "https://github.com/nessthehero"
-    },
-    {
-      "name": "Ryan Sprake",
-      "url": "https://github.com/eightdotthree"
-    },
-    {
-      "name": "Joseph Dillon",
-      "url": "https://github.com/jdillon522"
```

Comment:
![you-make-me-sad](https://cloud.githubusercontent.com/assets/4861202/24720762/32fdcf14-1a0c-11e7-8b83-3ed487e62a4e.gif)